src/components/layout/Navigation.tsx

```diff
@@ -46,7 +46,8 @@ const Navigator = (): JSX.Element | null => {
   });
 
   const isParentOwned =
-    (item?.creator?.id ?? parents?.first()?.creator?.id) === currentMember?.id;
+    currentMember &&
```

Comment:
Is this the breadcrumb fix ?

Reply:
I guess? At least it makes sense I think.